### PR TITLE
Changed logger to Pino

### PR DESCRIPTION
**Pull request by Scttpr**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
[Changer système de logs sur pino](https://trello.com/c/tiMvIb7r/351-changer-syst%C3%A8me-de-logs-sur-pino)

File: tests/models/role.mjs [hunk 1]
```diff
@@ -1,8 +0,0 @@
-import nanoid from 'nanoid';
```
Comment (Scttpr) [inline tests/models/role.mjs]: tests/models/role.mjs is unused and does not match Role model import. Probably a track of the past role management ?

Comment (Nainterceptor) [reply tests/models/role.mjs]: Maybe, you can remove that and if tests are still working, it was dead code.

Comment (Scttpr) [reply tests/models/role.mjs]: It was dead code

**Pull request merged**